content/en/get-started/deploy-your-app.md

```diff
@@ -103,150 +114,71 @@ strategies:
             weight: 100
 ```
 
-<table>
-<tr>
-<th>namespace-staging.yaml</th>
-<th>namespace-prod.yaml</th>
-</tr>
-<tr>
-<td>
-<pre>
-<code>
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: &#x3C;your-staging-namespace&#x3E;
-</code>
-</pre>
-</td>
-<td>
-<pre>
-<code>
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: &#x3C;your-prod-namespace&#x3E;
-</code>
-</pre>
-</td>
-</tr>
-</table>
 
+In your CD-as-a-Service config, you can create deployment strategies with as many steps as you want. In this example, the deployment config defines the following strategies:
 
+* `rolling`: deploy 100% of the app (staging deployment)
+* `trafficSplit`: 75% to the current version and 25% to the new version (prod deployment)
 
-{{% alert title="Important" color="warning" %}}
-For the first deployment of your app, Armory CD-as-a-Service automatically deploys the app to 100% of the cluster since there is no previous version. Subsequent deployments of your app follow the strategy steps defined in your deployment file.
-{{% /alert %}}
+The prod deployment requires a manual approval to begin deployment and another to continue deployment after the traffic split.
 
-## Deploy your app
+## 2. Deploy your app
+In order to see some deployment strategies at work, you'll need to deploy your app twice. The first time, CDaaS will skip any traffic splitting or manual approval steps in order to have a stable version fully deployed. The second deploy will demonstrate an advanced deployment strategy, routing only 25% of traffic to your new version, while routing the rest to the stable version.
 
-{{% alert title="Important" color="warning" %}}
-Armory CD-as-a-Service manages your Kubernetes deployments using ReplicaSet resources. During the initial deployment of your app, CD-as-a-Service deletes the underlying Kubernetes deployment object in a way that leaves behind the ReplicaSet and pods so that there is no actual downtime for your app. These are later deleted when the deployment succeeds.
-
-If your initial deployment fails, you should [manually delete]({{< ref "troubleshooting/tools#initial-deployment-failure-orphaned-replicaset" >}}) the orphaned ReplicaSet.
-{{% /alert %}}
-
-### Deploy first version
+### Deploy the first version
 
 1. Log in using the CLI.
 
    ```bash
    armory login
    ```
 
-   The CLI returns a `Device Code` and opens your default browser.  Confirm the code in your browser to complete the login process,.
+   The CLI returns a `Device Code` and opens your default browser.  Confirm the code in your browser to complete the login process.
 
-1. Start the deployment from the root of your directory.
+2. Start the deployment from the root of your directory.
 
```

Comment:
Numercial lists should all be:
```
1.
1.
1.
```

The list is compiled to the correct numbers. Using all 1s saves you from having to renumber if you insert another step into the top or middle of the list.